INSTALL

```diff
@@ -77,7 +73,8 @@ An optional installation of the glvis executable can be performed with
 
 Building with CMake
 ===================
-GLVis may also be configured using CMake to find dependencies.
+GLVis may also be configured using CMake to find dependencies. This is the only
+supported build method for Windows.
```

Comment:
Why is this? Are the cygwin and WSL builds (with GNU make) not working anymore?

Reply:
I'll add an annotation that this is for native Windows builds. I would assume WSL builds would fall under their respective Linux distribution's build instructions.

As for Cygwin, should we continue to support that build method after 4.1 is released? My impression is that Cygwin was a workaround for the POSIX/X11-specific bits of GLVis, and that now that those bits have been replaced with the new threaded server mode, we should encourage users to move to native Windows builds.

Reply:
I think we should discourage people from using Cygwin in favor of WSL.

Reply:
We don't need to do anything right now to ensure GLVis can be built under Cygwin -- if someone really wants to use it that way, then we can try to help, if something doesn't work.

For WSL: it will be good to make sure that works.